PanoramaPublic/resources/schemas/dbscripts/postgresql/panoramapublic-19.21-19.22.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+INSERT INTO panoramapublic.experimentannotations SELECT * FROM targetedms.experimentannotations;
+-- Update the PrimaryKey sequence
+SELECT SETVAL('panoramapublic.experimentannotations_id_seq', (SELECT MAX(id)+1 FROM panoramapublic.experimentannotations), false);
+
+INSERT INTO panoramapublic.journal SELECT * FROM targetedms.journal;
+-- Update the PrimaryKey sequence
+SELECT SETVAL('panoramapublic.journal_id_seq', (SELECT MAX(id)+1 FROM panoramapublic.journal), false);
+
+INSERT INTO panoramapublic.journalexperiment SELECT * FROM targetedms.journalExperiment;
```

Comment:
Do you have a plan for deleting the targetedms schema's copy of these tables? I didn't see anything here or on the TargetedMS module side of things.

Reply:
I was thinking of adding a SQL script in the targetedms module after LK 19.3 release. I would like to deploy and test LK 19.3 (with this module) on panoramaweb-dr without dropping the source tables in the targetedms schema.